site/src/components/WorkspaceActions/ActionCtas.tsx

```diff
@@ -0,0 +1,115 @@
+import Button from "@material-ui/core/Button"
+import { makeStyles } from "@material-ui/core/styles"
+import CloudQueueIcon from "@material-ui/icons/CloudQueue"
+import CropSquareIcon from "@material-ui/icons/CropSquare"
+import DeleteOutlineIcon from "@material-ui/icons/DeleteOutline"
+import HighlightOffIcon from "@material-ui/icons/HighlightOff"
+import PlayCircleOutlineIcon from "@material-ui/icons/PlayCircleOutline"
+import { FC } from "react"
+import { Workspace } from "../../api/typesGenerated"
+import { WorkspaceStatus } from "../../util/workspace"
+import { WorkspaceActionButton } from "../WorkspaceActionButton/WorkspaceActionButton"
+
+export const Language = {
+  start: "Start",
+  stop: "Stop",
+  delete: "Delete",
+  cancel: "Cancel",
+  update: "Update",
+}
+
+interface WorkspaceAction {
+  handleAction: () => void
+}
+
+export const StartButton: FC<WorkspaceAction> = ({ handleAction }) => {
+  const styles = useStyles()
+
+  return (
+    <WorkspaceActionButton
+      className={styles.actionButton}
+      icon={<PlayCircleOutlineIcon />}
+      onClick={handleAction}
+      label={Language.start}
+    />
+  )
+}
+
+export const StopButton: FC<WorkspaceAction> = ({ handleAction }) => {
+  const styles = useStyles()
+
+  return (
+    <WorkspaceActionButton
+      className={styles.actionButton}
+      icon={<CropSquareIcon />}
+      onClick={handleAction}
+      label={Language.stop}
+    />
+  )
+}
+
+export const DeleteButton: FC<WorkspaceAction> = ({ handleAction }) => {
+  const styles = useStyles()
+
+  return (
+    <WorkspaceActionButton
+      className={styles.actionButton}
+      icon={<DeleteOutlineIcon />}
+      onClick={handleAction}
+      label={Language.delete}
+    />
+  )
+}
+
+type UpdateAction = WorkspaceAction & {
+  workspace: Workspace
+  workspaceStatus: WorkspaceStatus
+}
+
```

Comment:
I think we should reconsider having the update button in this dropdown. It is not as cleanly related to workspace state as the other CTAs. It would be nice if the update functionality was congruent between the workspaces page and the detailed workspace view.

Reply:
If you think this is the best, I'm ok with that 👍